### DISCUSSION TITLE
12-02

### DISCUSSION THREAD

**Discussion by miniposi** [ChristmasTest/christmas#39] [Daily_Meeting]
### 🎅🏻 회의 내용

* 각자 해온 스토리북 발표
* 앞으로 할 일 분담

### ❄️ 오늘 할일

- [ ] 

### ⛄️ TMI

숙취가... 😵‍💫

**Comment by hanseulhee** [comment]
흐아 저도요 ......

**Comment by oauch** [comment]
다들 술쟁이 🍻
